screenpipe-events/src/custom_events/meetings.rs

```diff
@@ -0,0 +1,238 @@
+use crate::{send_event, subscribe_to_all_events};
+use anyhow::Result;
+use chrono::{DateTime, Utc};
+use futures::StreamExt;
+use serde::{Deserialize, Serialize};
+use serde_json;
+use std::collections::{HashMap, HashSet};
+use std::time::{Duration, Instant};
+
+const MEETING_APPS: &[&str] = &["zoom", "teams", "meet", "webex", "skype", "slack"];
+const MEETING_KEYWORDS: &[&str] = &[
+    "meeting",
+    "call",
+    "conference",
+    "joining",
+    "started",
+    "waiting room",
+    "lobby",
+    "participant",
+    "host",
+];
+const MEETING_END_PHRASES: &[&str] = &[
+    "meeting ended",
+    "call ended",
+    "left the meeting",
+    "host has ended",
+    "meeting will end",
+    "meeting has ended",
+];
+
+pub async fn poll_meetings_events() -> Result<()> {
```

Comment:
how confident about the accuracy of these detection? :D

Reply:
I tested it with zoom. I'll test with google meetings slack huddle, discord call, and teams and report back.